charts/stable/leantime/templates/_secrets.tpl

```diff
@@ -0,0 +1,20 @@
+{{/* Define the secrets */}}
+{{- define "leantime.secrets" -}}
+---
+
+apiVersion: v1
+kind: Secret
+type: Opaque
+metadata:
+  name: leantime-secrets
+{{- $leantimeprevious := lookup "v1" "Secret" .Release.Namespace "leantime-secrets" }}
+{{- $session_password := "" }}
+data:
+  {{- if $leantimeprevious}}
+  LEAN_SESSION_PASSWORD: {{ index $leantimeprevious.data "LEAN_SESSION_PASSWORD" }}
+  {{- else }}
+  {{- $session_password := randAlphaNum 32 }}
+  LEAN_SESSION_PASSWORD: {{ $session_password | b64enc | quote }}
```

Comment:
are you sure this is not something the users should have access to?
Remember, you can ONLY do this for things the users doesn't need to know!

Reply:
I've looked in their docs, they don't mention anywhere that is needed.
They also didn't have much docs about it either. Even the default they used is so far hidden, I doubt any user would find.
It's for salting only, according to this
https://github.com/Leantime/leantime/blob/master/config/configuration.sample.php